jobs/nats/templates/nats.cfg.erb

```diff
@@ -8,52 +8,39 @@ logtime: true
 log_file: /var/vcap/sys/log/nats/nats.log
 
 authorization {
-  DIRECTOR_PERMISSIONS: {
-    publish: [
-      "agent.*",
-      "hm.director.alert"
-    ]
-    subscribe: ["director.>"]
-  }
-
-  AGENT_PERMISSIONS: {
-    publish: [
-      "hm.agent.heartbeat._CLIENT_ID",
-      "hm.agent.alert._CLIENT_ID",
-      "hm.agent.shutdown._CLIENT_ID",
-      "director.*._CLIENT_ID.*"
-    ]
-    subscribe: ["agent._CLIENT_ID"]
-  }
-
-  HM_PERMISSIONS: {
-    publish: []
-    subscribe: [
-      "hm.agent.heartbeat.*",
-      "hm.agent.alert.*",
-      "hm.agent.shutdown.*",
-      "hm.director.alert"
-    ]
-  }
-
-  certificate_clients: [
-    {client_name: director.bosh-internal, permissions: $DIRECTOR_PERMISSIONS},
-    {client_name: agent.bosh-internal, permissions: $AGENT_PERMISSIONS},
-    {client_name: hm.bosh-internal, permissions: $HM_PERMISSIONS},
+  users = [
```

Comment:
Are you sure users can be specified twice (it also exists in the dynamic auth.conf). Also it looks like these users already get added here: https://github.com/cloudfoundry/bosh/pull/2391/files#diff-5cd6181b35dea1b09c5ef32d5ddaaf004157132204bc1436ec2e3293656bce11R47-R48

Reply:
During testing we saw that they can be specified twice and NATs will work. 
The ones you see on nats.cfg.erb are some sort of failsafe in the case that the nats-sync app cannot access the BOSH API when it is still initializing, making the list on the dynamic auth.conf being empty. But having them specified twice is not really necessary.